### PR TITLE
CEDS-1693 start page content

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: test/views/declaration/spec/UnitViewSpec.scala [hunk 1]
```diff
@@ -41,8 +44,7 @@ class UnitViewSpec extends UnitSpec with ViewMatchers {
   }
 }
 
-object UnitViewSpec {
-  val realMessagesApi: MessagesApi = {
-    new GuiceApplicationBuilder().injector().instanceOf(classOf[MessagesApi])
-  }
+object UnitViewSpec extends Injector {
+  val realMessagesApi: MessagesApi = instanceOf[MessagesApi]
+  val realAppConfig: AppConfig = instanceOf[AppConfig]
```
Comment (PatrykRudnicki) [inline test/views/declaration/spec/UnitViewSpec.scala]: I'm not sure that putting `realAppConfig` in UnitViewSpec is good idea. In practice it's not used in every spec but it'll be injected.

File: conf/application.conf [hunk 2]
```diff
@@ -145,6 +145,12 @@ contact-frontend {
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:6791/customs-declare-exports/start"
+
+  customsDeclarationsGoodsTakenOutOfEu = "https://www.gov.uk/guidance/customs-declarations-for-goods-taken-out-of-the-eu"
+  commodityCodes = "https://www.gov.uk/guidance/finding-commodity-codes-for-imports-or-exports"
+  relevantLicenses = "https://www.gov.uk/starting-to-export/licences"
+  serviceAvailability = "https://www.gov.uk/guidance/customs-declaration-service-service-availability-and-issues"
+  customsMovementsFrontend = "http://localhost:6796/customs-movements/start"
```
Comment (PatrykRudnicki) [inline conf/application.conf]: This must be overridden at least in `appConfigBase`, please prepare PR

File: test/views/declaration/spec/UnitViewSpec.scala (hunk 1 above)
Comment (MaciejRewera) [reply test/views/declaration/spec/UnitViewSpec.scala]: Changed to use mock. You're probably right.

File: conf/application.conf (hunk 2 above)
Comment (MaciejRewera) [reply conf/application.conf]: PR ready.

**Pull request merged**